src/util-decode-mime.h

```diff
@@ -197,10 +197,9 @@ typedef struct MimeDecParseState {
     uint8_t bvremain[B64_BLOCK];  /**< Remainder from base64-decoded line */
     uint8_t bvr_len;  /**< Length of remainder from base64-decoded line */
     uint8_t data_chunk[DATA_CHUNK_SIZE];  /**< Buffer holding data chunk */
-#ifdef HAVE_NSS
-    HASHContext *md5_ctx;
-    uint8_t md5[MD5_LENGTH];
-#endif
+    SCMd5 *md5_ctx;
```

Comment:
Is this an opague type? Wonder if we could not make it a pointer so prevent an allocation we'd do every time?

Reply:
Its an opaque type, but its a pointer returned from a rust constructor.